bazel/external/libcircllhist.BUILD

```diff
@@ -0,0 +1,15 @@
+cc_library(
+    name = "libcircllhist",
+    srcs = ["src/circllhist.c"],
+    hdrs = ["src/circllhist.h",
+            "src/circllhist_config.h", # Generated.
+    ],
+    includes = ["src"],
+    visibility = ["//visibility:public"],
+)
+genrule(
+    name = 'circllhist_config',
+    srcs = ['src/circllhist_config.h.in'],
+    outs = ['src/circllhist_config.h'],
+    cmd = 'cp $< $@',
```

Comment:
I have no idea on the Windows build. Assuming that it would come in future, what would be the alternative here?

Reply:
@jmillikin-stripe do you mind taking a quick look at this? Given how simple this library is (from build perspective) do we really even need the configure stuff for a native bazel build? Are there any targeted upstream fixes we could do to avoid this? cc @postwait